interface-definitions/service_pppoe-server.xml.in

```diff
@@ -70,19 +70,25 @@
             <children>
               <leafNode name="vlan-id">
                 <properties>
-                  <help>VLAN monitor for the automatic creation of vlans (user per vlan)</help>
-                  <constraint>
-                    <validator name="numeric" argument="--range 1-4096"/>
+                  <help>VLAN monitor for the automatic creation of single vlan</help>
+                  <valueHelp>
+                    <format>u32:1-4094</format>
+                    <description>VLAN monitor for the automatic creation of single vlan</description>
+                  </valueHelp>                  <constraint>
```

Comment:
oops - `<constraint>` moved into the wrong line.

Reply:
Sorry for this mistake. Done.